Content.Server/_RMC14/Xenonids/Hive/XenoHiveSystem.cs

```diff
@@ -38,16 +38,39 @@ public sealed class XenoHiveSystem : SharedXenoHiveSystem
 
     private readonly List<string> _announce = [];
     private readonly EntProtoId _defaultHive = "CMXenoHive";
+    private readonly Dictionary<EntityUid, List<TimedAnnouncement>> _pendingTimedAnnouncements = new();
```

Comment:
This should be tracked by adding a component to these entities or spawning new entities, not by keeping track of them in a system's dictionary like this
Otherwise you have to manually keep it in sync if the entity is deleted, the round restarts, etc